src/universal/modules/meeting/components/MeetingCheckInPrompt/CheckInQuestion.js

```diff
@@ -0,0 +1,261 @@
+import {css} from 'aphrodite-local-styles/no-important';
+import './CheckInQuestion.css';
+import {Editor, EditorState, getDefaultKeyBinding} from 'draft-js';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import FontAwesome from 'react-fontawesome';
+import 'universal/components/ProjectEditor/Draft.css';
+import withKeyboardShortcuts from 'universal/components/ProjectEditor/withKeyboardShortcuts';
+import withMarkdown from 'universal/components/ProjectEditor/withMarkdown';
+import Tooltip from 'universal/components/Tooltip/Tooltip';
+import appTheme from 'universal/styles/theme/appTheme';
+import ui from 'universal/styles/ui';
+import withStyles from 'universal/styles/withStyles';
+import {textTags} from 'universal/utils/constants';
+import entitizeText from 'universal/utils/draftjs/entitizeText';
+
+const iconStyle = {
+  fontSize: '1rem',
+  verticalAlign: 'middle',
+  marginLeft: '0.5rem'
+};
+
+const buttonStyle = {
+  cursor: 'pointer'
+};
+
+class CheckInQuestion extends Component {
+  static propTypes = {
+    canEdit: PropTypes.bool,
+    editorState: PropTypes.object,
+    setEditorState: PropTypes.func,
+    handleBeforeInput: PropTypes.func,
+    handleChange: PropTypes.func,
+    handleUpArrow: PropTypes.func,
+    handleDownArrow: PropTypes.func,
+    handleKeyCommand: PropTypes.func,
+    handleTab: PropTypes.func,
+    handleReturn: PropTypes.func,
+    keyBindingFn: PropTypes.func,
+    styles: PropTypes.object
+  };
+
+  state = {
+    isEditing: false
```

Comment:
draftjs gives us this for free. editorState.getSelection().hasFocus()

Reply:
womp